src/rgw/rgw_lua_background.h

```diff
@@ -0,0 +1,51 @@
+  #pragma once
+  #include <lua.hpp>
+  #include "common/dout.h"
+  #include "rgw_common.h"
+  
+  namespace rgw::lua {
+  
+  class Background {
+      protected:
+          std::unordered_map<std::string, int> rgw_map;
+          std::string rgw_script;
```

Comment:
might want to include headers for the type used here. IMHO, a header should be self-contained.